### PR TITLE
added checkout page

### PR DESCRIPTION
**Pull request by Ankur9669**
image

### CODE REVIEW

**Review by mehulsatardekar — APPROVED** [review]
the UI looks good and neat, didn't find any issues you can merge this branch

File: checkout.html [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                <img class="logo-image" alt="logo-image" src="images/neatUI.svg" />
+                <span class="logo-text">FiT</span>
+                BiT
+            </a>
+        </div>
+        <div class="nav-items-container-end">
+            <div class="login-search-bar-container">
+                <a href="logout.html" class="button button-primary">Logout</a>
+                <div class="input-box">
+                    <input type="text" class="input" value="" onchange="change(this)" required />
```
Comment (eakansh125) [inline checkout.html:37]: value attribute can be removed if not required

**Review by eakansh125 — APPROVED** [review]
Could not find a major syntactic issue. Clean code, ready to be merged from my side.

File: checkout.html (hunk 1 above)
Comment (Ankur9669) [reply checkout.html:37]: Ok i will see this

**Comment by kgajriaAgarwal** [comment]
everything seems to be good.can be merged !

**Review by pradipgharge — APPROVED** [review]
Code is clean, looks good to merge !

**Pull request merged**